ui/src/components/Tooltip.tsx

```diff
@@ -4,6 +4,7 @@ import React from "react";
 export interface TooltipProps extends MUITooltipProps {
     variant?: 'default' | 'wide' | 'fit';
     isSimple?: boolean;
+    classes?: any;
```

Comment:
Can you be more specific than `any`?

Reply:
I see it is any in our site a lot when using the classes